lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1387,6 +1390,49 @@ SPIRVExtInst *LLVMToSPIRVDbgTran::getSource(const T *DIEntry) {
   return Source;
 }
 
+template <class T>
+void LLVMToSPIRVDbgTran::genBuildIdentifierAndStoragePath(const T *DIEntry) {
+  static bool generated=false;
+  static uint64_t firstDWOId{0};
+  static std::string firstPathString{""};
+
+  // get information from LLVM IR
+  auto DWOId = DIEntry->getDWOId();
+  const std::string PathString = DIEntry->getSplitDebugFilename().str();
+
+  if (generated) {
+    // Ensure new information matches previous information
+    if (DWOId!=firstDWOId)
+      report_fatal_error(llvm::Twine("Multiple BuildIdentifier values are not allowed: ") +
```

Comment:
let's make it an assertion
it's our expected behavior to have a single build identifier and  storage path and assertion will express this contract, but if we are incorrect - users shouldn't get an error.
meanwhile assertions are good enough for debugging

Reply:
Changed into assertion.